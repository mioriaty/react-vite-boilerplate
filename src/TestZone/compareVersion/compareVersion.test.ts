import { compareVersion } from './compareVersion';

describe('compareVersion', () => {
  it('Parse version string to number then compare', () => {
    expect(compareVersion('1.0.0', '1.0.1')).toEqual(true);

    expect(compareVersion('1.0.0', '1.1.0')).toEqual(true);

    expect(compareVersion('1.0.0', '0.0.1')).toEqual(false);
  });
});
