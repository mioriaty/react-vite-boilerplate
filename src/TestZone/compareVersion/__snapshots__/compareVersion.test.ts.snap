// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`compareVersion > Parse version string to number then compare > false 1`] = `false`;

exports[`compareVersion > Parse version string to number then compare > true 1`] = `true`;

exports[`compareVersion > Parse version string to number then compare > true 2`] = `true`;

exports[`compareVersion > Parse version string to number then compare 1`] = `true`;

exports[`compareVersion > Parse version string to number then compare 2`] = `true`;

exports[`compareVersion > Parse version string to number then compare 3`] = `false`;
